services/monitoring/grafana/scripts/import.py

```diff
@@ -448,10 +440,8 @@ def main(foldername: str = "", overwrite: bool = True):
     print("**************** Add alerts *******************")
     # Finally we import the dashboards
     if foldername == "":
-        directoriesAlerts = glob.glob(
-            "./../provisioning/" + env.str("MACHINE_FQDN") + "/alerts/*"
-        )
-        directoriesAlerts += glob.glob("./../provisioning/allDeployments" + "/alerts/*")
+        directoriesAlerts = glob.glob("./../assets/alerts/*")
```

Comment:
TIP: `pathlib` incorporates the functionality you are using in `glob`. e.g.
```python
assets_dir = Path("./../assets/")
assert assets_dir.is_dir()

config_file_path = assets_dir /datasources2dashboards.yaml")
directories_alerts = list( (assets_dir / "alerts").glob("*") ] 
```

Reply:
Agree. This script (and others) need to be refactored but this is out of the scope of this PR and belongs to https://github.com/ITISFoundation/osparc-ops-environments/issues/189